content/authentication/troubleshooting-ssh/using-ssh-over-the-https-port.md

```diff
@@ -48,3 +60,14 @@ $ ssh -T git@{% data variables.command_line.codeblock %}
 > Hi <em>username</em>! You've successfully authenticated, but GitHub does not
 > provide shell access.
 ```
+
+## Known Hosts Requirements
```

Comment:
Formatting nit: we use title case (initial word capitalized, remaining words uncapitalized) for section titles.

Also, this section and title doesn't quite motivate what we're asking the user to do. So I would propose something like:

````
## Updating known hosts

The first time you interact with GitHub after switching to port 443, you may get a warning message
that the host wasn't found in `known_hosts`, or that it was found by another name.

```ShellSession
> The authenticity of host '[ssh.github.com]:443 ([140.82.112.36]:443)' can't be established.
> ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
> This host key is known by the following other names/addresses:
>     ~/.ssh/known_hosts:32: github.com
> Are you sure you want to continue connecting (yes/no/[fingerprint])? 
```

It is safe to answer `yes` to this question, assuming that the SSH fingerprint matches
one of GitHub's [published fingerprints](../keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints).
````

I'm not sure that we want to instruct users to preemptively add GitHub's keys to their `known_hosts`, since answering `yes` will do the right thing using whatever key their SSH client negotiates for.

Reply:
Thank you for the in-depth example and suggestion! 🚀 I've updated the content with this in dc03859c718c5821be04fc7e169a0f71f663931f.